src/components/MultiStepIndicator/__tests__/multiStepIndicator.jest.js

```diff
@@ -31,5 +31,34 @@ describe("MultiStepIndicator", () => {
       const tree = renderer.create(<MultiStepIndicator type={MultiStepIndicator.types.SUCCESS} steps={exampleSteps} />);
       expect(tree).toMatchSnapshot();
     });
+
+    it("Render correctly with clickable steps", () => {
```

Comment:
We may want to test that clicks are happening instead of the DOM structure.
What do you say?